compiler/src/dotty/tools/dotc/printing/Formatting.scala

```diff
@@ -168,7 +168,12 @@ object Formatting {
       case param: TypeParamRef =>
         s"is a type variable${addendum("constraint", ctx.typeComparer.bounds(param))}"
       case sym: Symbol =>
-        s"is a ${ctx.printer.kindString(sym)}${sym.showExtendedLocation}${addendum("bounds", sym.info)}"
+        val info =
+          if (ctx.gadt.bounds.contains(sym))
+            sym.info & ctx.gadt.bounds(sym)
```

Comment:
Do you need to do the intersection? Can you simply return `ctx.gadt.bounds(sym)`?

Reply:
I think so, we used to do something similar in PostTyper.